drivers/gpu/drm/drm_connector.c

```diff
@@ -1049,6 +1049,7 @@ static const struct drm_prop_enum_list drm_tv_mode_enum_list[] = {
 	{ DRM_MODE_TV_MODE_PAL_M, "PAL-M" },
 	{ DRM_MODE_TV_MODE_PAL_N, "PAL-N" },
 	{ DRM_MODE_TV_MODE_SECAM, "SECAM" },
+	{ DRM_MODE_TV_MODE_MONOCHROME, "MONOCHROME" },
```

Comment:
Nit: `MONOCHROME` is pretty long. Abbreviate to `MONO`?

I don't recall if the kernel command line has a max length, but it can get pretty unwieldy anyway just in normal usage.

Reply:
I guess the enum value should also be shortened to match the string?

Reply:
Actually not sure about changing the enum, as it could get confused with mono audio. MONOCHROME already appears in full in a spattering of symbol names (mostly private to particular VL2 and DRM drivers).

Reply:
Enum value within DRM ("MONOCHROME") or name (DRM_MODE_TV_MODE_MONOCHROME) ?

I have no issue with the enum name being that long as it's only ever in the source code. It's only that handling the long string becomes a headache at runtime.